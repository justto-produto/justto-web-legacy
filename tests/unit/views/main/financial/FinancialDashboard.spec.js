import FinancialDashboard from '@/views/main/financial/FinancialClientDashboard'
import { shallowMount } from '@vue/test-utils'

describe('Views > FinancialDashboard', () => {
  const wrapper = shallowMount(FinancialDashboard)

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
