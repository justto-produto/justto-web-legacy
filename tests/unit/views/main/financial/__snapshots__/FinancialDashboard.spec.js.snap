// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Views > FinancialDashboard renders correctly 1`] = `<jus-view-main />`;
